js/modules/swiperHelpers.js

```diff
@@ -0,0 +1,58 @@
+export function setSlidesState(){
+	const swiper = this;
+	const activeIndexes = getActiveIndexes.call(swiper);
+	const slideCollection = swiper.slides;
+	const mqlOver1920 = window.matchMedia("(min-width: 1920px)");
+
+	if(mqlOver1920.matches){
+		slideCollection.forEach(function(elem, index) {
+			if(activeIndexes.includes(index)){
+				elem.classList.remove("activeSlides");
+			}
+			else{
+				elem.classList.add("activeSlides");
+			}
+		});
+	}
+
+	mqlOver1920.addEventListener("change", on1920changeStateHandler);
+
+	function on1920changeStateHandler(){
+		if(mqlOver1920.matches){
+			swiper.on("slideChange", setSlidesState);
+		} else {
+			swiper.off("slideChange", setSlidesState);
+			cleanSlidesState.call(swiper);
+		}
+	}
+}
+
+function cleanSlidesState(){
+	const swiper = this;
```

Comment:
you can use parameters here instead fo `this`

Reply:
Done.